code/core/src/manager/components/preview/Toolbar.tsx

```diff
@@ -136,8 +136,17 @@ export const ToolbarComp = React.memo<ToolData>(function ToolbarComp({
   tabId,
   api,
 }) {
+  const id = useId();
+
   return tabs || tools || toolsExtra ? (
-    <Toolbar className="sb-bar" key="toolbar" shown={isShown} data-test-id="sb-preview-toolbar">
+    <Toolbar
+      className="sb-bar"
+      aria-labelledby={id}
+      key="toolbar"
+      shown={isShown}
+      data-test-id="sb-preview-toolbar"
+    >
+      <ScreenReaderLabel id={id}>Toolbar</ScreenReaderLabel>
```

Comment:
@valentinpalkovic FYI I've recently added a CSS util for visually hidden content which could simplify this ScreenReaderLabel code.

Reply:
See `code/core/src/theming/global.ts`